### DISCUSSION TITLE
Whether direct connection through IP address is allowed?

### DISCUSSION THREAD

**Discussion by thisleo** [alvr-org/ALVR#1005] [Q&A]
Is it possible to skip the client-side broadcast hostname and server-side link, but directly specify the server IP address to connect to the server

**Comment by zmerp** [comment]
The client has no UI so it is not possible to specify the server IP. However in the server it is possible to specify the client IP using the "Add client manually" button.

**Reply by thisleo** [reply]
In the latest version (V18.2.1), it is possible to specify the IP address of the ALVR server by running the command line at the command prompt
`adb shell am startservice -n "com.polygraphene.alvr/.ChangeSettings" --es "targetServers" "10.10.10.10"`

**Reply by zmerp** [reply]
I'm pretty sure you can't do this anymore since ALVR v12 or v13.

**Reply by thisleo** [reply]
I need this feature, what should I do

**Reply by zmerp** [reply]
Sorry, there is nothing we can do at the moment.
